torch/onnx/symbolic.py

```diff
@@ -863,15 +863,29 @@ def replication_pad(g, input, padding):
 replication_pad3d = replication_pad
 
 
-@parse_args('v', 'is')
```

Comment:
This shouldn't be deleted

Reply:
I guess there has been updates to upsample that for the argument output_size it could be either a tensor or a int list. Thus the annotation becomes `@parse_args('v', 'v')` which can be ignored.

Reply:
That's right, as @BowenBao mentioned adding @pars_args('v','v') will be redundant.